recipes/myproxyclient/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "MyProxyClient" %}
+{% set version = "1.4.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "5b7ad3cc3c3cd71595a9b0278a51c138699e229d47a41f2e2add72fbc12f58cd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  entry_points:
+    - myproxyclient = myproxy.script:main
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - pyopenssl
+
+test:
+  imports:
+    - myproxy
+    - myproxy.test
+    - myproxy.utils
+  commands:
+    - myproxyclient --help
+
+about:
+  home: https://github.com/cedadev/MyProxyClient
+  license: BSD 3-clause
```

Comment:
Could we please add or ask to have a `MANIFEST.in` with the license file listed upstream? This way `sdist`s will include the license and we can more easily package the license file in our `conda` package. Thus remaining compliant with the terms of the BSD 3-Clause license. This is not a blocker to get this merged, but it would be nice to get the ball rolling.